inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -1440,6 +1440,29 @@ void FuseMultipleIdentitiesPass::run() {
     }
 }
 
+void PowerBroadcastPass::run() {
```

Comment:
Is this correct name of the pass? Does it describe what this pass does? I don't think it has anything to do with broadcasting.

Reply:
No, It's not correct. Now, I am rewriting this implementation.